insert_attachments_id_token_btn_plugin/insert_attachments_id_token.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="plugin" group="editors-xtd" version="4.0" method="upgrade">
+    <name>plg_editors-xtd_insert_attachments_id_token_btn</name>
+    <version>4.0.4</version>
+    <creationDate>November  9, 2024</creationDate>
+    <author>Jonathan M. Cameron</author>
+    <copyright>(C) 2007-2024 Jonathan M. Cameron. All rights reserved.</copyright>
+    <license>http://www.gnu.org/licenses/gpl-3.0.html GNU/GPL</license>
+    <authorEmail>jmcameron@jmcameron.net</authorEmail>
+    <authorUrl>https://github.com/jmcameron/attachments/</authorUrl>
+    <description>ATTACH_INSERT_ATTACHMENTSID_TOKEN_BUTTON_PLUGIN_DESCRIPTION</description>
+    <namespace path="src">JMCameron\Plugin\EditorsXtd\InsertAttachmentsIdToken</namespace>
+    <files>
+        <folder>src</folder>
+        <folder plugin="insert_attachments_id_token">services</folder>
+    </files>
+	<languages>
+		<language tag="en-GB">language/en-GB/en-GB.plg_editors-xtd_insert_attachmentsid_token.ini</language>
```

Comment:
The files here have wrong filenames. They should be "*_attachmets_id_*.ini" instead of "*_attachmentsid_*.ini".